packages/myst-to-typst/src/index.ts

```diff
@@ -363,10 +363,10 @@ const handlers: Record<string, Handler> = {
   legend: captionHandler,
   captionNumber: () => undefined,
   crossReference(node: CrossReference, state, parent) {
-    if (node.remote) {
```

Comment:
xrefs to other pages are still considered `remote` - we only want to do this if there is a `remoteBaseUrl`, i.e. they are xrefs to a totally separate project.